src/11.numberformat.js

```diff
@@ -723,7 +724,7 @@ function ToRawFixed (x, minInteger, minFraction, maxFraction) {
     let idx,
 
         // We can pick up after the fixed formatted string (m) is created
-        m   = Number.prototype.toFixed.call(x, maxFraction),
+        m = decimalAdjust('round', x, -maxFraction).toFixed(maxFraction),
```

Comment:
we should try to align with the spec text: https://tc39.github.io/ecma402/#sec-torawfixed, and if the spec is incorrect, we can attempt to fix it, but we should not diverge from the spec by creating a new abstract.